### PR TITLE
Update ScriptPod default image

### PR DESCRIPTION
**Pull request by scme0**
# Background

#project-k8s-agent

Upgrade the default script pod image to use a slimmed down image that only has the required dependencies.

[sc-70343]

### CODE REVIEW

File: source/Octopus.Tentacle/Kubernetes/KubernetesPodContainerRegistry.cs [hunk 1]
```diff
@@ -36,7 +36,7 @@ public async Task<string> GetContainerImageForCluster()
 
             var tag = tagVersion?.ToString(2) ?? "latest";
 
-            return $"octopuslabs/k8s-workertools:{tag}";
+            return $"docker.packages.octopushq.com/octopusdeploy/kubernetes-agent-tools-base:{tag}";
```
Comment (APErebus) [inline source/Octopus.Tentacle/Kubernetes/KubernetesPodContainerRegistry.cs:39]: Are we pushing this image to dockerhub yet? Or are we going to wait until we've tested it more?

Comment (scme0) [reply source/Octopus.Tentacle/Kubernetes/KubernetesPodContainerRegistry.cs:39]: I was thinking we'd wait until we've tested it and then just before we release we can push it to dockerhub.

Comment (APErebus) [reply source/Octopus.Tentacle/Kubernetes/KubernetesPodContainerRegistry.cs:39]: yep, I think that's a fair call. Can you put a ticket in Shortcut so we don't forget to do this?

**Comment by scme0** [comment]
New shortcut story to review this image before we release the agent EAP: https://app.shortcut.com/octopusdeploy/story/76030/review-the-new-script-pod-container-image

**Pull request merged**